modules/btcpay/src/Server/Webhook.php

```diff
@@ -34,8 +34,8 @@ public function __construct(string $baseUrl, string $apiKey, ClientInterface $cl
 	 */
 	public function ensureWebhook(string $storeId): void
 	{
-		// Check if we have an existing webhook, if so, just cancel now
-		if (null === $this->getCurrent($storeId, $this->configuration->get(Constants::CONFIGURATION_BTCPAY_WEBHOOK_ID))) {
```

Comment:
looking at the getCurrent() method the problem here likely was that the check was `===` null instead of `!==`  null but yes the old empty() check also works

Reply:
Pretty change on my part. I did get a `''` returned at some point, so the `empty`-checked seemed to be in place for a good reason.